pandas/tests/series/indexing/test_getitem.py

```diff
@@ -71,7 +71,7 @@ def test_getitem_unrecognized_scalar(self):
     def test_getitem_negative_out_of_bounds(self):
         ser = Series(["a"] * 10, index=["a"] * 10)
 
-        msg = "index -11 is out of bounds for axis 0 with size 10"
+        msg = "index -11 is out of bounds for axis 0 with size 10|index out of bounds"
```

Comment:
can you use a "|".join pattern

Reply:
I don't feel super strong here but I prefer the pipe pattern for 2 options